db/migrate/20201221212948_add_time_zone_to_patients.rb

```diff
@@ -0,0 +1,29 @@
+include PatientHelper
```

Comment:
Could you update the date of this migration so that it is the most recent? Schema date will update as well as part of this change.

Reply:
@kylietmo Updated migration date